core/src/integrationTest/java/amino/run/multidm/MultiDMTestCases.java

```diff
@@ -173,143 +173,140 @@ private MicroServiceSpec createMultiDMTestSpec(String... names) throws Exception
      * @return package name for the given short DM name.
      */
     private String getPackageName(String name) throws Exception {
-        if (name.equals("DHT")) {
-            return "amino.run.policy.dht";
-        }
-        if (name.equals("ConsensusRSM")) {
-            return "amino.run.policy.replication";
-        }
         if (name.equals("AtLeastOnceRPC")) {
             return "amino.run.policy.atleastoncerpc";
         }
         if (name.equals("CacheLease")) {
             return "amino.run.policy.cache";
         }
```

Comment:
Many 'if' conditions. Can we add 'switch' case instead of 'if' condition?

Reply:
This all needs to be rewritten.  It's a mess.  In the mean time I just followed what is there.